deploy/terraform-aws/main.tf

```diff
@@ -179,8 +179,8 @@ EOF
   tags = local.k8s_cluster_tags
 }
 
-resource "aws_iam_role_policy" "k8s_node" {
-  name = "k8s-node-${var.cluster_name}"
+resource "aws_iam_role_policy" "k8s_node_k8s_policy" {
+  name = "k8s-node-${var.cluster_name}-k8s"
```

Comment:
nit: is the addition of `-k8s` after cluster name necessary since it is prefixed with the same?

Reply:
Disregard this comment, since you have broken out the policy into three separate I now see the logic behind it.